packages/atlas/src/views/global/YppLandingView/YppRewardSection.tsx

```diff
@@ -56,24 +64,42 @@ export const YppRewardSection: FC = () => {
           data-aos-offset="80"
           data-aos-easing="atlas-easing"
         >
-          <BenefitsCardButton
-            variant={rewardMultiplier === 1 ? 'primary' : 'tertiary'}
-            onClick={() => setRewardMultiplier(1)}
-          >
-            &lt;5K subscribers
-          </BenefitsCardButton>
-          <BenefitsCardButton
-            variant={rewardMultiplier === 1.5 ? 'primary' : 'tertiary'}
-            onClick={() => setRewardMultiplier(1.5)}
-          >
-            &lt;5-25K subscribers (1.5x)
-          </BenefitsCardButton>
-          <BenefitsCardButton
-            variant={rewardMultiplier === 3 ? 'primary' : 'tertiary'}
-            onClick={() => setRewardMultiplier(3)}
-          >
-            &gt;25K subscribers (3x)
-          </BenefitsCardButton>
+          {tiers.map((tier, idx, tierArray) => {
+            const isFirstTier = idx === 0
+            const isLastTier = idx === tierArray.length - 1
+            if (isFirstTier) {
+              return (
+                <BenefitsCardButton
+                  variant={rewardMultiplier === tier.multiplier ? 'primary' : 'tertiary'}
+                  key={tier.minimumSubscribers}
+                  onClick={() => setRewardMultiplier(tier.multiplier)}
+                >
+                  &lt;{formatNumber(tierArray[idx + 1].minimumSubscribers)} subscribers
+                </BenefitsCardButton>
+              )
+            }
+            if (isLastTier) {
+              return (
+                <BenefitsCardButton
+                  variant={rewardMultiplier === tier.multiplier ? 'primary' : 'tertiary'}
+                  key={tier.minimumSubscribers}
+                  onClick={() => setRewardMultiplier(tier.multiplier)}
+                >
+                  &gt;{formatNumber(tier.minimumSubscribers)} subscribers
+                </BenefitsCardButton>
+              )
+            }
+            return (
+              <BenefitsCardButton
+                variant={rewardMultiplier === tier.multiplier ? 'primary' : 'tertiary'}
+                key={tier.minimumSubscribers}
+                onClick={() => setRewardMultiplier(tier.multiplier)}
+              >
+                &lt;{formatNumber(tier.minimumSubscribers)}-{formatNumber(tierArray[idx + 1].minimumSubscribers)}{' '}
```

Comment:
I think we don't need `&lt;` for this middle tier? We want this to say `100-200 subscribers` instead of `<100-200`, right?